dev/django.Dockerfile

```diff
@@ -11,8 +11,10 @@ ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
 RUN curl -L -o /tmp/shapeworks.zip https://github.com/SCIInstitute/ShapeWorks/releases/download/v6.2.1/ShapeWorks-v6.2.1-linux.zip && \
-    unzip -d /opt /tmp/shapeworks.zip && rm /tmp/shapeworks.zip
-ENV PATH $PATH:/opt/ShapeWorks-v6.2.1-linux/bin
+    unzip -d /tmp /tmp/shapeworks.zip && \
+    mv /tmp/ShapeWorks-v6.2.1-linux /opt/shapeworks && \
```

Comment:
Will we need to manually update the version of shapeworks as  more releases are made?

Reply:
Yes.  We should eventually write a script that checks the latest version on github releases.

Reply:
Yeah, sadly we will need to update the version here in the Dockerfile and in the deploy_ec2_worker.sh script. Something to centralize or standardize those version numbers would be good to have.